tests/robustness/model/history.go

```diff
@@ -488,36 +489,17 @@ func (h History) Len() int {
 
 func (h History) Operations() []porcupine.Operation {
 	operations := make([]porcupine.Operation, 0, len(h.operations))
-	maxTime := h.lastObservedTime()
 	for _, op := range h.operations {
 		// Failed requests don't have a known return time.
 		if op.Return == -1 {
-			// Simulate Infinity by using last observed time.
-			op.Return = maxTime + time.Second.Nanoseconds()
+			// TODO(henrybear327): add reasoning here
```

Comment:
You can just leave previous comment `// Simulate infinity for failed requests`

Reply:
Done as requested!